iulianIachimov/src/main/java/dataCollectionTask/Models/Item.java

```diff
@@ -1,19 +1,50 @@
 package dataCollectionTask.Models;
 
 public class Item {
-  public long id;
-  public String title;
-  public Type type;
+  private final long id;
+  private String title;
+  private Type type;
+  private double price;
   private static long idCounter;
 
-  public Item(String title, Type type) {
+  public Item(String title, Type type, double price) {
     this.id = ++idCounter;
```

Comment:
let's have the same pattern for all ids, let's use UUID as in examples above

Reply:
Done